lib/chef/azure/core/bootstrap_context.rb

```diff
@@ -51,6 +45,13 @@ def config_content
           end
 
           client_rb << @config[:user_client_rb] + "\r\n" unless @config[:user_client_rb].empty?

```

Comment:
So there are still things before this -- `user_client_rb` be the very first thing in this function instead of `client_rb = ""`? Otherwise those things will get overridden by the client.rb file when they are specified explicitly outside of it, and we want the opposite behavior.